com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -141,76 +400,150 @@ public SceneSwitchProgress SwitchScene(string sceneName, LoadSceneMode loadScene
                 throw new Exception($"{nameof(NetworkConfig.EnableSceneManagement)} flag is not enabled in the {nameof(NetworkManager)}'s {nameof(NetworkConfig)}. Please set {nameof(NetworkConfig.EnableSceneManagement)} flag to true before calling this method.");
             }
 
-            if (s_IsSwitching)
+            // Return scene event already in progress if one is already in progress... :)
+            if (s_IsSceneEventActive)
             {
-                if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
-                {
-                    NetworkLog.LogWarning("Scene switch already in progress");
-                }
-
-                return null;
+                return new SceneEventProgress(null, SceneEventProgressStatus.SceneEventInProgress);
```

Comment:
Is this going to cause weirdness where the scene name we return is different to the scene name that was supplied, and there's not really any notification that this is happening?